frontend/src/components/LandingPage/LandingPage.js

```diff
@@ -3,20 +3,53 @@ import './LandingPage.css'
 const LandingPage = () => {
     return (
         <div className="home-body-div">
-            <div className="home-top-div">
-                <div className="logo-div">
-                    <img src='/logo-dark.png' className='dark-logo' />
+            <div className="top-bottom-div">
+
+                <div className="home-top-div">
+                    <div className="logo-div">
+                        <img src='/logo-dark.png' className='dark-logo' />
+                    </div>
+                    <div className="user-buttons-div">
+                        <button className='demo-btn jello-wiggle button__shine__short'>Demo User</button>
+                        <button className='login-btn green-btn jello-wiggle button__shine__long__green'>Log In</button>
+                        <button className='signup-btn green-btn jello-wiggle button__shine__long__green'>Sign Up</button>
+                    </div>
                 </div>
-                <div className="user-buttons-div">
-                    <button className='demo-btn jello-wiggle button__shine__short'>Demo User</button>
-                    <button className='login-btn green-btn jello-wiggle button__shine__long__green'>Log In</button>
-                    <button className='signup-btn green-btn jello-wiggle button__shine__long__green'>Sign Up</button>
+                <div className='home-bottom-div'>
+                    <p><span id="bulk">Jell-O</span> is a variety of gelatin desserts, puddings, and no-bake cream pies</p>
+                    <p><span id="bulk">Trello</span> is a popular, simple-to-use collaboration tool that enables you to organize projects and everything related to them into boards</p>
+                    <p id="get-both-text">When you mix both... you get <span id="bulk">Jello</span>, a cuter Trello</p>
+                </div>
+                <div className="coder-profiles">
+
+                    <div className="coder">
+                        <a target="_blank" href="github.com">
+                            <img src="/jello9.png" className="coder-img jello-wiggle"></img>
```

Comment:
Similarly, I believe these files should be moved back into `/public/static/`, and the sources changed to match.